go.mod

```diff
@@ -51,23 +50,19 @@ require (
 	go.uber.org/multierr v1.4.0 // indirect
 	go.uber.org/zap v1.13.0 // indirect
 	golang.org/x/lint v0.0.0-20191125180803-fdd1cda4f05f // indirect
-	golang.org/x/net v0.0.0-20191209160850-c0dbc17a3553
+	golang.org/x/net v0.0.0-20200202094626-16171245cfb2
 	golang.org/x/oauth2 v0.0.0-20191202225959-858c2ad4c8b6
 	golang.org/x/sys v0.0.0-20191210023423-ac6580df4449 // indirect
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
-	gonum.org/v1/gonum v0.6.0 // indirect
 	google.golang.org/api v0.7.0
 	google.golang.org/appengine v1.6.5 // indirect
 	google.golang.org/genproto v0.0.0-20191206224255-0243a4be9c8f // indirect
-	google.golang.org/grpc v1.25.1 // indirect
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
-	gopkg.in/yaml.v2 v2.2.7
-	k8s.io/api v0.17.1
-	k8s.io/apimachinery v0.17.1
-	k8s.io/client-go v0.17.1
-	k8s.io/code-generator v0.17.1
-	k8s.io/gengo v0.0.0-20191010091904-7fa3014cb28f // indirect
+	gopkg.in/yaml.v2 v2.2.8
+	k8s.io/api v0.18.0
```

Comment:
Hey @ricardomaraschini - in the openshift/origin PR they are referencing 1.18.0-rc.1

Were you aware?  Or was there a reason you did not go with that ?

Reply:
No, I was no aware of it. Is this `rc` a release candidate to 1.18(0.18.0)? Could you point me to the PR you are referring to?

Reply:
Well, I see when I tried rc.1, subsequent updates for openshift/api,client-go,library-go bumped it to v0.18.0 anyway ;-) 

So *nevermind* :-)